systemtests/tests/autochanger/testrunner

```diff
@@ -61,6 +79,17 @@ for i in $(seq ${NUMBER_OF_POOLS}); do
   done
 done
 
+#resetting only 2 devices to `autoselect = yes` to check that devices with `autoselect = no` don't get used.
```

Comment:
This change now only allows two jobs to run on two drives in parallel. Before, four jobs were writte on four tapes which makes sense to test also.
I think the functionality that devices set to autoselect=no are not selected is already tested in the multiplied-devices test and so does not need to be tested here.

Reply:
The whole idea of this change is that we want to be able to label tapes even if all drives are busy.
For that, we want to free one drive by disabling the autoselect, then be able to label a tape in the free drive, and 
after the label is done reenable the drive so that it will be used for  backups again.

So we would need something like this:
- all drives are busy with jobs the whole time
- while this is the case, we set autoselect to no one drive.
- wait until the drive is not in use anymore
- label a new tape in this drive
- set autoselect to yes again
- check if the next jobs will use the drive again
- It might make sense to do this in an own test.

Reply:
Thanks!

done!